DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionInterceptor.swift

```diff
@@ -6,6 +6,8 @@
 
 import Foundation
 
+// TODO: RUM-3470 Add tests to `URLSessionInterceptor`
```

Comment:
💡 `URLSessionInterception` has no tests coverage. I will add tests in next PR. Here, we're merging against feature branch.